config/protocol.testnet.neofs.yml

```diff
@@ -0,0 +1,86 @@
+ProtocolConfiguration:
+  Magic: 735783775
+  MaxTraceableBlocks: 2102400
+  InitialGASSupply: 52000000
+  SecondsPerBlock: 15
+  MemPoolSize: 50000
+  StandbyCommittee:
+  - 0337f5f45e5be5aeae4a919d0787fcb743656560949061d5b8b05509b85ffbfd53
+  - 020b86534a9a264d28b79155b0ec36d555ed0068eb1b0c4d40c35cc7d2f04759b8
+  - 02c2efdc01181b0bc14fc19e0acb12281396c8c9ffe64458d621d781a1ded436b7
+  - 026f9b40a73f29787ef5b289ac845bc43c64680fdd42fc170b1171d3c57213a89f
+  - 0272350def90715494b857315c9b9c70181739eeec52d777424fef2891c3396cad
+  - 03a8cee2d3877bcce5b4595578714d77ca2d47673150b8b9cd4e391b7c73b6bda3
+  - 0215e735a657f6e23478728d1d0718d516bf50c06c2abd92ec7c00eba2bd7a2552
+  StateRootInHeader: false
+  ValidatorsCount: 7
+  SeedList:
+  - morph1.t5.fs.neo.org:50333
+  - morph2.t5.fs.neo.org:50333
+  - morph3.t5.fs.neo.org:50333
+  - morph4.t5.fs.neo.org:50333
+  - morph5.t5.fs.neo.org:50333
+  - morph6.t5.fs.neo.org:50333
+  - morph7.t5.fs.neo.org:50333
+  VerifyBlocks: true
+  VerifyTransactions: true
+  P2PSigExtensions: true
+  NativeActivations:
```

Comment:
No hardforks set in the testnet?

Reply:
Yes, it's pretty new, Aspidochelone+. Technically, we'll have to set `Aspidochelone: 0` after some B fork, but for now it works just fine without any additional lines.